topn-limit-push-down.md

```diff
@@ -0,0 +1,127 @@
+---
+title: TopN and Limit Operator Pushdown
+summary: Learn the implementation of TopN and Limit operator pushdown.
+category: performance
```

Comment:
```suggestion
```

Reply:
> > It's better to replace `collation` with `ordering rule` to avoid misunderstandings with the collation of sql.
> 
> Is there any ref on this one? Like, how would you choose between `ordering rule` and `sorting rule`?

No, `collation` generally refers to the collation of the character set. You can choose from other options that you think are more appropriate.